app/assets/templates/pin-index.html

```diff
@@ -1,32 +1,32 @@
-<div class="background container-fluid">
+<div class="container-fluid background">
```

Comment:
which background are we talking about? use more specific class names: when you'll be looking at your css file later (or when someone else will) you'll see a .background class but you'll never know what it refers to. Use .pin-background or main-background, etc.